mingw.thread.h

```diff
@@ -30,22 +37,9 @@
 #include <process.h>
 #include <ostream>
 
-#ifdef _GLIBCXX_HAS_GTHREADS
-#error This version of MinGW seems to include a win32 port of pthreads, and probably    \

```

Comment:
I would like to preserve this, and move it to the point where the classes are imported into std::

Reply:
New commit uses `#pragma message` to provide a non-fatal redundancy warning.